backend/src/models/Project.ts

```diff
@@ -2,6 +2,8 @@ import { model, ObjectId, Schema } from "mongoose";
 import { ITech } from "./Tech";
 
 export interface IProject {
+  id?: string;
```

Comment:
nit: Would prefer to keep only one or the other ID field in the type definition here. The likeDislikeHelpers appear to be projecting the $id correctly, so I'm not sure what might be using _id instead of "id" to cause the typescript compiler to complain.